CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+[1.7.0] - 2022-02-14
+--------------------
+
+### New features
+
+- Support all bonding options
```

Comment:
```suggestion
- NetworkManager provider: Support all available bonding modes and options
```

IMHO it is important to highlight that this is for NM only.

Reply:
I updated. thanks.